google/resource_compute_instance_group_manager.go

```diff
@@ -201,7 +201,7 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 			"wait_for_instances": &schema.Schema{
 				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  true,
+				Default:  false,
```

Comment:
Calling out for future-Paddy--this doesn't require a migration or anything because this hasn't been released yet.